bskyweb/templates/base.html

```diff
@@ -213,6 +213,7 @@
     }
 
     /* NativeDropdown component */
+    .radix-dropdown-item:focus,
```

Comment:
Just being more specific here, since now we're using Radix directly. I'd like to remove the other class once `NativeDropdown` is phased out.